server/models.py

```diff
@@ -1373,3 +1373,21 @@ def delete(self):
         db.session.delete(self)
         db.session.commit()
         return self
+
+class Job(db.Model):
+    """A background job."""
+    statuses = ['queued', 'running', 'finished']
+
+    id = db.Column(db.Integer, primary_key=True)
+    updated = db.Column(db.DateTime(timezone=True), onupdate=db.func.now())
+    status = db.Column(db.Enum(*statuses, name='status'), nullable=False)
+
+    # The user who started the job.
+    user_id = db.Column(
+        db.Integer, db.ForeignKey('user.id')
+    )
+    user = db.relationship('User')
```

Comment:
Would it be useful to have a row for what course it belongs to as well? That way all staff members can see what jobs for a course have been run?